sliceguard/embeddings.py

```diff
@@ -116,6 +134,86 @@ def generate_image_embeddings(
     return embeddings
 
 
+def _extract_probabilities_image(model, feature_extractor, col_name="image"):
```

Comment:
See above. Reuse the extract_image_embeddings method if it significantly safes redundant code.